src/snowflake/cli/api/utils/models.py

```diff
@@ -1,9 +1,15 @@
+from __future__ import annotations
+
 import os
+from typing import Any, Dict
 
 
-class EnvironWithDefinedDictFallback(dict):
+class EnvironWithDefinedDictFallback(Dict):
```

Comment:
why? `dict` refers to the built-in dictionary class, it seemed fine to me before.

Reply:
The signature of update_from_dict was using it, so I thought it makes sense to be consistent. If that's a problem, I can change it back.